docs/affinity.md

```diff
@@ -0,0 +1,86 @@
+
+# Affinity
+
+For every service, there is a set of service workers and a set of affine
+Hyperbahn relays (the subset of the relay ring that is responsible for traffic
+to and from the given service).
+The minimum viable Hyperbahn fully connects the affine workers and relays,
```

Comment:
I think you have "affine" modifying the wrong noun here, shouldn't it be: "connects the workers to the affine relays" ?